### PR TITLE
update xmanager colabs

### PR DESCRIPTION
**Pull request by rsadiq**
Update SKAI Xmanager Colab Notebooks

### CODE REVIEW

File: src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb [hunk 1]
```diff
@@ -69,17 +69,17 @@
         "#@title Please run this cell first!\n",
         "\n",
         "#@markdown Specify the parameters to set up your Colab notebook. They should be the same that the ones used during the initialization of the Colab kernel\n",
-        "#############################################\n",
-        "### CODE SETTING - ENVIRONMENT ACTIVATION ###\n",
-        "#############################################\n",
+        "###################################################################################\n",
+        "### CODE SETTING - ENVIRONMENT ACTIVATION and Installation of required packages ###\n",
```
Comment (ambaha1) [inline src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb:73]: If not need for the virtual environment, please remove any reference to it in the notebook

File: src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb [hunk 2]
```diff
@@ -103,50 +111,54 @@
...
         "DATASET_NAME = \"skai_dataset\" #@param {type:\"string\"}\n",
-        "DATASET_GCP_PATH = \"gs://skai-data/hurricane_ian\" #@param {type:\"string\"}\n",
-        "GCP_OUTPUT_DIR = \"gs://skai-data/experiments/skai_train_vizier\" #@param {type:\"string\"}\n",
+        "DATASET_LABELED_TRAIN = \"gs://bucket/path/to/examples/labeled_train_examples.tfrecord\" #@param {type:\"string\"}\n",
+        "DATASET_VALIDATION = \"gs://bucket/path/to/examples/labeled_train_examples.tfrecord\" #@param {type:\"string\"}\n",
+        "GCP_OUTPUT_DIR = \"gs://bucket/path/to/output/\" #@param {type:\"string\"}\n",
+        "GOOGLE_CLOUD_BUCKET_NAME = \"\" #@param {type:\"string\"}\n",
+        "EXPERIMENT_NAME = \"skai_train_xamanager\" #@param {type:\"string\"}\n",
         "NUM_EPOCHS = 10 #@param {type:\"integer\"}\n",
         "ACCELERATOR = \"V100\" #@param [\"V100\",\"T4\"]\n",
```
Comment (ambaha1) [inline src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb:125]: If the use of TPU has not been tested, please remove this option for the moment.

**Review by ambaha1 — CHANGES_REQUESTED** [review]
Thank you Rizwan for submitting this PR.
I left few comment in the code, please apply the according change.
Overall, my main observations at this stage are:
- revise in the near future the relationship between the different colab pipeline notebooks, this one focusing only the training using xmanager, vs others performing the other sections of the pipeline (examples generation, zs, labelling)
- check the need of the virtual environment for this colab or not, and the use of the management of the parameters/variables across the colab environment.
- always good to add a description to the PR, this helps the reviewer to understand the content and logic of the changes pushed.

**Review by ambaha1 — CHANGES_REQUESTED** [review]
Thank you Rizwan. I left few comments for changes requests at this stage.
Looking forward and overall we will need:
- revise the connection between the different colab, and see in which extent specify each colabs for each section of the pipeline or merge this xmanager colab into the main one.
- if the first option choosen, review the way the skai project parameters/variables and the python environment flow between the colabs.
- make sure you can input a description in the PR, as it helps the reviewer to better undertand the changes that are pushed.

Feel free once the changes we discused are implemented, to resubmit for merging.

**Review by ambaha1 — CHANGES_REQUESTED** [review]
Thank you Rizwan. I left few comments for changes requests at this stage.

Looking forward and overall we will need:
- revise the connection between the different colab, and see in which extent specify each colabs for each section of the pipeline or merge this xmanager colab into the main one.
- if the first option choosen, review the way the skai project parameters/variables and the python environment flow between the colabs.
- make sure you can input a description in the PR, as it helps the reviewer to better undertand the changes that are pushed.

Feel free once the changes we discused are implemented, to resubmit for merging.

File: src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb (hunk 1 above)
Comment (rsadiq) [reply src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb]: ok

**Review by rsadiq — COMMENTED** [review]
Hi @ambaha1 The requested changes have been updated in the recent comits

**Review by ambaha1 — APPROVED** [review]
Thank you Rizwan for the changes. It looks good for now.
On top of the previous overall comments left and going forward, the use of the virtual environment need to be clarified for xmanager commands.

File: requirements.txt [hunk 3]
```diff
@@ -5,7 +5,7 @@ fiona
 gcsfs
 geopandas>=0.8
 google_apitools
-google-cloud-aiplatform>=1.35.0
+google-cloud-aiplatform>=1.48.0
```
Comment (jzxu) [inline requirements.txt:8]: @rsadiq Hi Rizwan, can you explain why this library version needs to be bumped?

Comment (rsadiq) [reply requirements.txt:8]: Hi @jzxu , it was because of installation issue of the specific version and its compatibility with the other packages.

**Pull request merged**